### DISCUSSION TITLE
长按倍速播放

### DISCUSSION THREAD

**Discussion by linger040831** [hooke007/mpv_PlayKit#236] [Q&A 问答] [answered]
请问有长按倍速播放的相关指令吗？比如长按 RIGHT   键1.5倍速播放。

**Comment by zhongfly** [comment, marked as answer]
mpv自身不支持这样细分的快捷键，只能通过脚本完成

起源： https://github.com/hooke007/MPV_lazy/discussions/183
最简单的只能对单一键强化的脚本 https://github.com/Natural-Harmonia-Gropius/mpv_config/blob/5d517e312ef631d693edbc2eeb002c9c71d5b66d/scripts/pressaction.lua
全面扩展变成 https://github.com/Natural-Harmonia-Gropius/InputEvent

个人修改版：https://github.com/zhongfly/InputEvent 增加条件判断功能、支持外部配置

**Reply by hooke007** [reply]
如上，要么在自己脚本里自定义，要么用第三方脚本实现，原生input.conf不支持“长按”这种类似模拟手势的按键定义

**Reply by linger040831** [reply]
谢谢回答！

**Reply by systemoutprintlnhelloworld** [reply]
英雄!!我自己写了半天没弄出来

**Reply by 351492384fate** [reply]
这个脚本的双击动作好像和mpv.net有冲突，原版mpv倒是没问题
update:鼠标双击及以上的多击都会有问题，比如Usage里的第一个示例的鼠标左键单击以暂停，双击以全屏就不行，不知道为什么

**Reply by zhongfly** [reply]
> 这个脚本的双击动作好像和mpv.net有冲突，原版mpv倒是没问题

我一直都是用mpv.net的，上述脚本我都用过，没有问题啊。
update:鼠标双击在mpv.net的确有问题，其他快捷键双击都是没有问题的

**Reply by natural-harmonia-gropius** [reply]
https://github.com/mpvnet-player/mpv.net/issues/533 mpv.net 鼠标左键没有 keyup

**Comment by 2607627233** [comment]
只是长按倍速的话 evafast比较好用
